exercises/list-ops/canonical-data.json

```diff
@@ -0,0 +1,147 @@
+{
+  "#": [
+    "Though there are no specifications here for dealing with large lists,",
+    "implementers may add tests for handling large lists to ensure that the",
+    "solutions have thought about performance concerns."
+  ],
+  "append": {
+    "description": "append entries to a list and return the new list",
+    "cases": [
+      {
+        "description": "empty lists",
+        "list1": [],
+        "list2": [],
+        "expected": []
+      },
+      {
+        "description": "empty list to list",
+        "list1": [],
+        "list2": [1,2,3,4],
+        "expected": [1,2,3,4]
+      },
+      {
+        "description": "non-empty lists",
+        "list1": [1,2],
+        "list2": [2,3,4,5],
+        "expected": [1,2,3,4,5]
```

Comment:
good catch.  I think at the last minute i decided that having a both arrays be larger than 1 element seemed like it would enhance the test a bit but I only updated `list1` and not the `expected`